src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.exception.InvalidFruitException;
+import core.basesyntax.model.Fruit;
+import java.util.List;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public Fruit get(String name) {
+        return FruitStorage.fruits.stream()
+                .filter(fruit -> fruit.getName().equals(name))
+                .findFirst()
+                .get();
```

Comment:
Here you can get NullPointerException
```suggestion
                .orElseThrow(() -> new InvalidFruitException("Can't find fruit: " + name));
```